pyproject.toml

```diff
@@ -108,6 +108,7 @@ requests = "*"
 rocrate = "*"
 Routes = "*"
 schema-salad = "!=8.3.20220721194857"  # https://github.com/common-workflow-language/schema_salad/issues/575
+python-slugify = "*"
```

Comment:
Sorry for nitpicking, can you please move it up so it's in alphabetical order?

Reply:
Sorry about that!